src/data/list/chain.lean

```diff
@@ -64,6 +64,10 @@ theorem chain_split {a b : α} {l₁ l₂ : list α} : chain R a (l₁ ++ b :: l
 by induction l₁ with x l₁ IH generalizing a;
 simp only [*, nil_append, cons_append, chain.nil, chain_cons, and_true, and_assoc]
 
+@[simp] theorem chain_append_singleton_cons (r : α → α → Prop) (a b c : α) (l m : list α) :
```

Comment:
```suggestion
@[simp] theorem chain_append_cons_cons (r : α → α → Prop) (a b c : α) (l m : list α) :
```

Reply:
That was actually the planned name, must have gotten confused halfway through.